composer.json

```diff
@@ -0,0 +1,18 @@
+{
+   "name": "ifixit/popsql",
+   "description": "A simple, objective approach to conditionally constructing MySQL SELECT statements.",
+   "homepage": "https://github.com/iFixit/PopSQL",
+   "type": "library",
+   "license": "Zlib",
+   "require": {
+      "php": ">=5.6"
+   },
+   "autoload": {
+      "classmap": [
```

Comment:
👀 maybe we should use psr4?

Reply:
I'd have to add a namespace, right? I was trying not to touch the source in this pull